content/blog/eng/protected-friday.md

```diff
@@ -0,0 +1,126 @@
+---
+authors: ["engineering"]
+comments: true
+date: "2023-04-17"
+draft: false
+share: true
+categories: [Venerdì Protetto]
+title: "Venerdì Protetto"
+languageCode: "en-US"
+type: "post"
+twitterImage: '/images/logo_engineering.png'
+toc: false
+---
+
+# Venerdì Protetto
+
+## What is it all about?
+
+At Facile.it, we strongly believe in sharing knowledge and experiences. Thus, we dedicate one Friday a month to expanding our knowledge and learning among colleagues in the IT department. During these days called Venerdì Protetto (i. e. Protected Friday), we welcome speakers and participate in talks on IT-related topics. Speakers who bring to life Venerdì Protetto are colleagues or external guests.

```

Comment:
"Speakers who bring to..." sounds strange to me... is it idiomatic? Or it's a transliteration from Italian?

Reply:
Idiomatic: to make active, lively. You are right, should have written: ...bring VP to life...
It's not necessary, though. Deleted to avoid confusion.